src/python/deepgnn/graph_engine/snark/converter/writers.py

```diff
@@ -39,24 +41,22 @@ def __init__(
             suffix (int): file suffix in the name of binary files
             skip_node_sampler(bool): skip generation of node alias tables
             skip_edge_sampler(bool): skip generation of edge alias tables
+            skip_timestamps(bool): skip generation of dynamic graph data
```

Comment:
missing is_temporal and watermark docstring. Also is skip_timestamps even a parmaeter?

Reply:
added docstring